nilearn/plotting/find_cuts.py

```diff
@@ -54,6 +54,13 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
     img = check_niimg_3d(img)
     data = _safe_get_data(img)
 
+    # Retrieve optional mask
+    if mask_img is not None:
+        mask_img = check_niimg_3d(mask_img)
+        mask = _safe_get_data(mask_img)
```

Comment:
Do we really have the guarantee that img and mask_img are in the same space? We should at least check it, and at best raise an error or do a resampling if not.